src/signal_manager.sh

```diff
@@ -0,0 +1,30 @@
+. "$KW_LIB_DIR/kwio.sh" --source-only
+
+# This functions prints the default message when interrupting kw
+function default_interrupt_handler() {
+  say $'\n'$SEPARATOR
+  say Exiting...
+}
+
+# This function adds a new signal handler to an arbitrary signal
+#
+# @command The handler to be called when a signal is catched
+# @signals The signals to attach to this handler
+function signal_manager() {
+  local command="$1"
+  shift
+  local -a signals="$@"
+
+  if [ -z "$command" -o -z "${signals[@]}" ]; then
```

Comment:
Hmmm... I think this `-z "${signals[@]}"`reduces the value of having a `signal_manager` function. How about this:
1. Check if `"${signals[@]}"` is empty.
1.2. Empty: Define the default signals.
1.3. Not empty: Use the parameter value

Also, it might be a good idea to validate if we have a valid signal.

Reply:
Done in the last update.